docs/graphs/graph-sdk-example.md

```diff
@@ -0,0 +1,289 @@
+LET doc = (FOR d in groceryItems FILTER d.name == @name RETURN d)[0]
+FOR user IN 1..1 INBOUND doc groceryOrders
+    FOR item IN 1..1 OUTBOUND user groceryOrders
+        COLLECT i=item.name WITH COUNT INTO c
+        FILTER i != @name
+        SORT c DESC
+        // LIMIT 2
+        RETURN {"item": i, "count": c}   // item -> number of users who purchased it
+                                         // count -> number of times it was purchased
+```
+
+</TabItem>
+
+</Tabs>
+
+The `@name` is a bind variable that can be set to any item name. In this example we can set it to `peanut butter`. First we are filtering by item name and then collecting all the users who purchased that item. Then we are collecting all the items that were purchased by those users. We are filtering out the item that we are looking (`peanut butter`) for and sorting the result by the number of times it was purchased.
+
+### Create Query Worker
+
+We can create a Query Worker on Macrometa GDN using console, javascript SDK, CLI or using the REST API.
+Exported Query Worker can be found in the `sample-data` folder of this [repository](https://github.com/Macrometacorp/macrometa-graph-example). It is possible to import the query worker using console. (`import-query.json`)
+
+
+After we have created the Query Worker, we can start using it using the following code:
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+
+```js
+const jsc8 = require("jsc8");
+
+client = new jsc8({
+  url: "https://play.paas.macrometa.io",
+  apiKey: "xxxxx",
+  fabricName: "_system",
+});
+
+let itemCustomerSelected = "peanut butter";
+
+async function queryGraph() {
+  // Query has bind variables that we need to provide as a second argument
+  response = await client.executeRestql("grocery-recomendations", {
+    name: itemCustomerSelected,
+  });
+  console.log(response.result);
+}
+queryGraph();
+```
+
+</TabItem>
+
+</Tabs>
+
+### Query result
+
+If customer selected `peanut butter` item, the result will following recommendation:
+
+```json
+[
+  { item: 'jelly', count: 2 },
+  { item: 'napkins', count: 2 },
+  { item: 'strawberry', count: 2 },
+  { item: 'banana', count: 1 },
+  { item: 'milk', count: 1 },
+  { item: 'celery', count: 1 },
+  { item: 'bread', count: 1 },
+  { item: 'white claw', count: 1 },
+  { item: 'kiwi', count: 1 },
+  { item: 'ketchup', count: 1 },
+  { item: 'apple', count: 1 }
+]
+```
+
+First recommendation according to this query is jelly which is purchased by 2 users. Second recommendation is napkins which is purchased by 2 users. Third recommendation is strawberry which is purchased by 2 users.
+
+## CRUD operations on graph
+
+It is possible to create, read, update and delete vertices and edges in the graph using javascript SDK.
+
+Full example can be found in this [repository](https://github.com/Macrometacorp/macrometa-graph-example).
+
+## Example Data Set
```

Comment:
I'm confused again. I thought the sample data was supposed to be in the repo that was linked near the top, but now it is also here??

What I've done in previous examples where I have data is use a C8QL query to add docs to an existing collection. This might be something to do with the SDK. Show users how to use the SDK to:

- Create collections of different types.
- Add data to the collections.
- Create graphs using the collection.
- Query the collection. 
- Create query workers.
- Run query workers, especially ones that use bind parameters. Bonus points if some need bind parameters and some don't.